src/panel/top-bar.tsx

```diff
@@ -134,6 +150,47 @@ export default function Topbar() {
         }
         <Message>{state.context.message}</Message>
       </CollapseSegment>
+      <FileButtons>
+        {
```

Comment:
nit: you dont need the brackets

Reply:
For the `// @ts-ignore` line

Reply:
can we fix the type instead

Reply:
types are from Storybook, I could try bumping the package. Seems a bit sketchy. Let me see if that works tho.

Reply:
Nah it's filthy. Upgrading to Storybook 6 does resolve the type problem but breaks everything else.